spec/Section 4 -- Introspection.md

```diff
@@ -147,6 +147,9 @@ type __Type {
 
   # should be non-null for NON_NULL and LIST only, must be null for the others
   ofType: __Type
+
+  # should be non-null for INPUT_OBJECT only
```

Comment:
I have complaints about this section more generally, but for consistency we should have a "must be null for the others" clause here

Reply:
I've explicitly NOT added this because it prevents future expansion of `oneOf` to other types (e.g. the Object type).